harvester/utils/ckan_utils.py

```diff
@@ -762,8 +762,12 @@ def create_ckan_resources(metadata: dict) -> list[dict]:
                 continue
             resource["url"] = dist[url_key]
             # set mimetype if provided or discover it
-            if "mimetype" in dist:
-                resource["mimetype"] = dist["mediaType"]
+            if "mediaType" in dist:
+                # this means it's an ISO record from mdtranslator
+                if dist["mediaType"] == "placeholder/value":
+                    resource["mimetype"] = guess_resource_format(dist[url_key])
+                else:
+                    resource["mimetype"] = dist["mediaType"]
             else:
                 resource["mimetype"] = guess_resource_format(dist[url_key])
 
```

Comment:
It's weird to have duplicated lines. I would maybe have a case checking for the DCAT-US type case (`mediaType in dist` and not `placeholder/value`), and then let the default otherwise be guessing.